HW03-Import-and-Graph.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "HW03"
+output: github_document
+author: Laura Cespedes Arias
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,warning=FALSE)
+```
+
+## Package loading
+```{r load libraries}
+library("ggplot2")
+library("readr")
+library("MASS")
+```
+
+I decided to do a simple figure with some data I am analyzing right now. The data is not public yet, so I uploaded a subset of our original data table to make the graph. Specifically, this is data of above ground carbon biomass [Baccini et al 2017](https://science.sciencemag.org/content/early/2017/09/27/science.aam5962?versioned=true) corresponding to stations where we counted migratory birds in Colombia. For each station, we also assigned a habitat category. I wanted to explore if the habitat category described fairly well the variation in above ground biomass, which is a variable that is supposed to describe the amount of "tree" biomass. Therefore, is expected that forested habitats would have higher values than more open habitats. 
+
+
+## Data import
+First, I imported the data table that I previously uploaded to GitHub:
+
+```{r}
+biomass_URL <- "https://raw.githubusercontent.com/lcespedesarias/HW03/master/CarbonBiomass_byHabitat.csv"
```

Comment:
Yes! Absolutely a good point. I am still trying to get used to GitHub so I was a little confused about this. It is not necessary to pull the file from the web because the file is already in the project folder. Thanks for pointing this out!